recipes/lue/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -e
+
+
+if [ $(uname) == Linux ]; then
+  # Kindly guide to conda's OpenGL...
+  PLATFORM_OPTIONS="-D OpenGL_GL_PREFERENCE=LEGACY \
+  -D OPENGL_opengl_LIBRARY:PATH=${BUILD_PREFIX}/${HOST}/sysroot/usr/lib64/libGL.so \
+  -D OPENGL_gl_LIBRARY:PATH=${BUILD_PREFIX}/${HOST}/sysroot/usr/lib64/libGL.so "
```

Comment:
Is this the right way to do this @isuruf @beckermr?

Reply:
This seems perfectly reasonable to me. Of course since CMake is turing complete, the actual CMakeLists.txt can do whatever they want!

Reply:
Sytle wise, I would use a bash array though!

Reply:
thanks, I also incorporated the suggestion